src/components/ChatApp/MessageListItem/MessageListItem.react.js

```diff
@@ -86,6 +86,25 @@ class MessageListItem extends React.Component {
       this.onYouTubePlayerReady,
     );
   }
+
+  render() {
```

Comment:
is it just me or there are in fact 2 render functions in the same component. Did you run the code before submitting?